### DISCUSSION TITLE
Porting CosmosOS to Raspberry Pi or ARM

### DISCUSSION THREAD

**Discussion by RetroAndDev** [CosmosOS/Cosmos#1742] [Ideas]
Hello,
I submit the idea of porting the CosmosOS compiler to ARM for the Raspberry Pi.
I hope you can do it
See you soon!
Bye

**Comment by valentinbreiz** [comment]
Hello @ArtGo-Devloppper, @jp2masa was working on it a long time ago. All of his work must be outdated / with a lot of conflics. Supporting a different CPU architecture is a huge amount of work and without contributors it will stay the way it is...

You can find a pull request here: https://github.com/CosmosOS/Cosmos/pull/495
